### ISSUE TITLE
Update the pretrained VGG16 or not?

### ISSUE THREAD

**Issue by o0t1ng0o** [lyndonzheng/F-LSeSim#9]
Hi, @lyndonzheng 
I have a question about the optimization of pretrained VGG16 when training the Learned SeSim model.
As shown in the following code, the learning rate multiplies zero.
I am wondering whether the pretrained VGG16 will update its parameters during training? 
https://github.com/lyndonzheng/F-LSeSim/blob/e092e62ed8a2f51f3661630e1522ec2549ec31d3/models/sc_model.py#L109

Thank you in advance.

**Comment by lyndonzheng** [comment]
Hi @o0t1ng0o, we did not update weights of pre-trained VGG16. Indeed, we only select suitable features using the contrastive learning from the pre-trained VGG model. We tried to update all the weights using the contrastive learning, while the performance is unstable under current design.

**Comment by o0t1ng0o** [comment]
Thank you for your prompt reply.
